src/main/webapp/app/shared/util/utils.ts

```diff
@@ -65,8 +65,8 @@ export const round = (value: any, exp?: number) => {
     value = +value;
     exp = +exp;
 
-    if (isNaN(value) || !(exp % 1 === 0)) {
-        return NaN;
+    if (isNaN(value) || exp % 1 !== 0) {
```

Comment:
I am not sure if `round(NaN)` returning `0` makes sense.

The bug stems from a division by zero in the task component and we should fix the division by zero happening there instead of passing `NaN` into the round function and expecting it to return 0